packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -361,6 +361,19 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "gateio", pair: "amplusdt" },
       { type: "cryptowatch", exchange: "bitfinex", pair: "amplusd" }
     ]
+  },
+  XAUUSD: {
```

Comment:
Are there any financial contracts that are currently using this identifier? If not then no need to add XAUUSD to this file.

Reply:
Do we generally not preemptively add to this file once an identifier has been approved?

Reply:
Generally no but there's no problem preemptively doing so. However if we're going to use the `ExpressionPriceFeed` here then we'll want the XAUUSD one